examples/Example1_BasicReadings/Example1_BasicReadings.ino

```diff
@@ -12,33 +26,38 @@ uint8_t deviceAddress = kQwiicUltrasonicDefaultAddress; // 0x2F
 
 void setup()
 {
-    // Start serial
-    Serial.begin(115200);
-    Serial.println("Qwiic Ultrasonic Example 1 - Basic Readings");
+  // Start serial
+  Serial.begin(115200);
+  Serial.println("Ultrasonic Distance Sensor Example 1 - Basic Distance Sensing");
```

Comment:
IMO the print here should match the file name ("Basic Readings" in this case, rather than "Basic Distance Sensing")